crates/holochain/src/core/ribosome/host_fn/block_agent.rs

```diff
@@ -92,94 +90,6 @@ mod test {
         let _response3: ZomeCallResponse = conductor.call(&alice, "try_cap_claim", cap_for).await;
     }
 
-    #[tokio::test(flavor = "multi_thread")]
-    #[cfg(feature = "slow_tests")]
-    async fn zome_call_get_block() {
```

Comment:
Deleting this test because it fails now after the refactor, and it doesn't test what it's supposed to test, and I plan to move it anyway as an integration test in the Holochain crate, along with other tests.